package.json

```diff
@@ -42,22 +39,46 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.0",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-private-methods": "^7.16.11",
+    "@babel/plugin-proposal-private-property-in-object": "^7.16.7",
     "@babel/preset-env": "^7.16.11",
-    "@babel/preset-typescript": "^7.12.13",
+    "@babel/preset-react": "^7.16.7",
+    "@babel/preset-typescript": "^7.16.7",
+    "@nrwl/cli": "13.7.2",
+    "@nrwl/cypress": "13.7.2",
+    "@nrwl/devkit": "13.7.2",
```

Comment:
@gromon42 Why did you switch the dependencies into the devDependencies?
NX needed this to build the production, isn't optimal but we didn't any solution atm